DiceKit/Die.swift

```diff
@@ -116,3 +116,14 @@ extension Die: ExpressionType {
 public func == (lhs: Die, rhs: Die) -> Bool {
     return lhs.sides == rhs.sides
 }
+
+// MARK: - Debug
+extension Die: CustomDebugStringConvertible {
+    
+    public var debugDescription: String {
+        get {
+            return "I am a Die! I have \(self.sides) sides!"
```

Comment:
Yeah. I have some code right now that will show expression components separated by their operator, and d notation for the die. But for debug I think it might be better if the type was more clear. 

Also has me thinking about some stuff like "if the multiplier of a MultiplicationExpression is a Constant, print it like this. Else, print it in ()" etc. Then we would practically have reverse string code! :stuck_out_tongue: